### PR TITLE
[ADD] l10n_in_reports: add profit and loss report (India)

### PR DESCRIPTION
**Pull request by nash-odoo**
In this PR, added the profit and loss report with opening and closing stock.

**task**-4143190

### CODE REVIEW

File: content/applications/finance/fiscal_localizations/india.rst [hunk 1]
```diff
@@ -550,3 +550,13 @@ The :guilabel:`GSTR-3` report contains different sections:
 
    .. image:: india/gst-gstr-3-report.png
       :alt: GSTR-3 Report
+
+Profit and Loss (IN) report
+---------------------------
+
+This is a :guilabel:`Profit & Loss` report that displays the balances for **Opening Stock** and
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/india.rst:557]:
```suggestion
This is a :guilabel:`Profit and Loss` report that displays the balances for **Opening Stock** and
```

File: content/applications/finance/fiscal_localizations/india.rst [hunk 2]
```diff
@@ -550,3 +550,13 @@ The :guilabel:`GSTR-3` report contains different sections:
...
+**Closing Stock**. It helps users using Continental Accounting to accurately determine the
+Cost of Goods (i.e Opening Stock + Purchases During the Period - Closing Stock).
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/india.rst:559]:
```suggestion
**Closing Stock**. It helps users using Continental accounting to accurately determine the cost of
goods (i.e :guilabel:`Opening Stock` + purchases during the period - :guilabel:`Closing Stock`).
```

**Review by dade-odoo — CHANGES_REQUESTED** [review]
Hey @nash-odoo thanks for the addition! Just a few minor capitalization/formatting comments. Once they're resolved I'll pass you on to the final review and r+ :)

**Review by dade-odoo — APPROVED** [review]

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hi @nash-odoo thank you for your work! It looks like the PNG file wasn’t compressed. Could you please take care of that when you get a chance? Thanks!

**Comment by nash-odoo** [comment]
Hi @auva-odoo, I compressed the image.
Thanks for the review :blush:

**Review by auva-odoo — APPROVED** [review]
Thank you @nash-odoo! You can r+ whenever you're ready :slightly_smiling_face: 
@robodoo delegate+

**Comment by nash-odoo** [comment]
@robodoo r+

**Pull request closed without merging**